src/main/java/Invokers/ApiClient.java

```diff
@@ -1201,7 +1201,7 @@ public <T> ApiResponse<T> execute(Call call, Type returnType) throws ApiExceptio
 			responseCode = String.valueOf(response.code());
 			status = response.message();
 
-			logger.debug("Network Response :\n" + json.serialize(response.body()));
+			logger.debug("Network Response :\n" + json.serialize(response.headers()));
```

Comment:
I just ran into this issue, and I was about to create a PR to fix this line. Is there an anticipated day for the September release?

Reply:
It will get released by 21st sept.